File: Ejercicio/JavaScript/Binario/BinarioPalidromo.js
let binPal = "10101";
let invertido = binPal.split("").reverse().join("");

if (binPal === invertido) {
  console.log(`El número binario ${binPal} es palíndromo.`);
} else {
  console.log(`El número binario ${binPal} no es palíndromo.`);
}

File: Ejercicio/GO/Binario/BinarioPalidromo.go
package main

import "fmt"

func main() {
	binario := "10101"
	invertido := ""

	for i := len(binario) - 1; i >= 0; i-- {
		invertido += string(binario[i])
	}

	if binario == invertido {
		fmt.Printf("El número binario %s es palíndromo.\n", binario)
	} else {
		fmt.Printf("El número binario %s no es palíndromo.\n", binario)
	}
}
